src/components/BlogPost.js

```diff
@@ -1,44 +1,41 @@
 import React from 'react'
 import styled from 'styled-components'
 
-const BlogPostWrapper = styled.div`
-  border: 3px solid #000;
-  width: 50%;
+import Date from './Date'
+import Image from './Image'
+import Paragraph from './Paragraph'
+import Tag from './Tag'
+import Title from './Title'
+
+// Possible Colors
```

Comment:
do you still need this comment here? I noticed that color declarations were removed here.